contrib_bots/bot_lib.py

```diff
@@ -33,12 +35,18 @@ def is_legal(self):
         else:
             return True
 
+    def show_error_and_exit(self):
+        logging.error(self.error_message)
+        sys.exit(1)
```

Comment:
There's other code, besides `run.py`, that will eventually be running this `bot_lib.py` code.  For example, the Flask server, or even the Zulip server.  I think it would be really unfortunate if the entire Zulip server shut down just because a bot ran into an issue(!)

At some point (probably not in this diff though), we should find a way to do error handling that doesn't involve `sys.exit`.